### DISCUSSION TITLE
Code splitting of configureStore

### DISCUSSION THREAD

**Discussion by gajraj-gan** [reduxjs/redux-toolkit#4075] [Q&A] [answered]
I have seen that rtk 2.0 as been released, we currently use configureStore along with createSlice, is there way to use split the slice into different bundles now? Most of the slices are only used in some specific part of the application and need not be in the main chunk. 

Thank you !!

**Comment by EskiMojo14** [comment, marked as answer]
In RTK 2.0 we added `combineSlices` which is designed to allow for lazy reducer injection.

Currently only the [API reference page](https://redux-toolkit.js.org/api/combineSlices) exists but I'm in the process of putting together a full [example](https://github.com/reduxjs/redux-toolkit/pull/4064) and [adding it to the code splitting page](https://github.com/reduxjs/redux/pull/4665).

**Reply by gajraj-gan** [reply]
Okay, thanks !!
